pcap/stream.go

```diff
@@ -152,6 +153,9 @@ func (f *tcpFlow) reassembledWithIgnore(ignoreCount int, sg reassembly.ScatterGa
 
 		f.currentParser = nil
 		f.currentParserCtx = nil
+
+		// FIXME; is this going to be too high data-rate?
+		telemetry.Error("parser", err)
```

Comment:
Did you intend to address this? Looks like `telemetry.RateLimitError` might be an alternative.

Reply:
Fixed in https://github.com/akitasoftware/akita-cli/pull/155/commits/bafd054e083b8a473730d71ca4b58ff43abd12af